ES/ESdd045.xml

```diff
@@ -286,54 +277,63 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <item xml:id="a1">
                            <locus target="#134v">f. 134v</locus>
                            <desc>: Purchase note.</desc>
```

Comment:
it would be preferable to add `@type` to the desc of additions, if this is among the adjustments you would like to make